src/views/home/components/EditionCard.vue

```diff
@@ -68,11 +70,17 @@ import EditionIcons from '@/components/cues/edition-icons.vue';
 export default class EditionCard extends Vue {
     @Prop() public edition!: EditionInfo;
 
+    private get thumbnailSourceExists(): boolean {
+        return (undefined !== this.edition
+                 && undefined !== this.edition.thumbnail ) ;
+    }
+
     private get thumbnailSource(): string | undefined {
-        return this.edition.thumbnail
```

Comment:
Change this to `return this.edition?.thumbnail?.thumbnailUrl` and it will work as expected. No need for a thumbnailSourceExists method at all.